src/test/unit/views/et-daily-list.test.ts

```diff
@@ -137,18 +137,18 @@ describe('ET Daily List page', () => {
 
     it('should display Case Name without Case Sequence Indicator', () => {
         const cell = htmlRes.getElementsByClassName('govuk-table__cell');
-        const x = 9;
+        const x = 8;
```

Comment:
Need to remove CaseType from the test data for unit tests and end2end tests too

Reply:
OK, Kian will do